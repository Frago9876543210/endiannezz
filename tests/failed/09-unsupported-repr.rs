use endiannezz::Io;

#[derive(Io)]
#[endian(big)]
#[repr(C)]
enum Foo {
    Bar,
    Baz,
}

fn main() {}
