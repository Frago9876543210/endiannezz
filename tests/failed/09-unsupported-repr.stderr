error: Unsupported repr type
 --> $DIR/09-unsupported-repr.rs:5:1
  |
5 | / #[repr(C)]
6 | | enum Foo {
7 | |     Bar,
8 | |     Baz,
9 | | }
  | |_^
